portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -1,18 +1,39 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.gson.Gson;
+
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
+    public static ArrayList<String> commentArray = new ArrayList<>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("I spend a lot of my free time playing League of Legends or practicing on one of my four guitars!");
+    response.setContentType("application/json;");
+    
+    //comments
```

Comment:
Do you notice that if you fetch the comments multiple times, you end up with all your comments duplicated?
It looks to me like that will happen since every time doGet runs you add all of the messages to the same commentArray. I would recommend either creating the commentArray every time as a local variable in doGet or adding all of the comments once to the static array using static initialization.